test/parallel/test-buffer-slow.js

```diff
@@ -39,21 +39,24 @@ try {
   assert.strictEqual(e.name, 'RangeError');
 }
 
-// Should work with number-coercible values
-assert.strictEqual(SlowBuffer('6').length, 6);
-assert.strictEqual(SlowBuffer(true).length, 1);
-
-// should throw with invalid length
+// should throw with invalid length type
+const bufferInvalidTypeMsg = common.expectsError({
+  code: 'ERR_INVALID_ARG_TYPE',
+  type: TypeError,
+  message: /^The "size" argument must be of type number/,
+}, 4);
+assert.throws(() => SlowBuffer(), bufferInvalidTypeMsg);
```

Comment:
Please do not mix `assert.throws()` and `common.expectsError()`. It will work but it is unnecessary overhead. Instead, either only use `common.expectsError()` with a predefined error object such as:

```js
const bufferInvalidTypeMsg = {
  code: 'ERR_INVALID_ARG_TYPE',
  type: TypeError,
  message: /^The "size" argument must be of type number/,
};
common.expectsError(() => SlowBuffer(), bufferInvalidTypeMsg);
// ...
```
Or
```js
const bufferInvalidTypeMsg = {
  code: 'ERR_INVALID_ARG_TYPE',
  name: 'TypeError [ERR_INVALID_ARG_TYPE]',
  message: /^The "size" argument must be of type number/,
};
assert.throws(() => SlowBuffer(), bufferInvalidTypeMsg);
// ...
```